core/src/main/java/ch/cyberduck/core/http/HttpConnectionPoolBuilder.java

```diff
@@ -180,7 +180,6 @@ public HttpClientBuilder build(final Proxy proxy, final TranscriptListener liste
                 new BackportWindowsNegotiateSchemeFactory(null) :
                 new SPNegoSchemeFactory())
             .register(AuthSchemes.KERBEROS, new KerberosSchemeFactory()).build());
-        configuration.setDnsResolver(new CustomDnsResolver());
```

Comment:
What is the intention with removing the custom resolver implementation?

Reply:
Removing unnecessary code? This shouldn't be necessary anymore as the operating system should be the one making the decision of which IP version is used.

Reply:
For context, this came from #13399 which specifically was added for IPv6 preference.

Reply:
I am hesitant as this will cause us to use two different resolver implementations in our [generic connection setup](https://github.com/iterate-ch/cyberduck/blob/e255b0c424d6e24a528fafdbe2acc525c48c8b8c/core/src/main/java/ch/cyberduck/core/LoginConnectionService.java#L39) and for the HTTP connection pool. In the best case this will only (but still) result in different error messages missing localization.

Reply:
I don't understand (and obviously I'm not familiar with the entire codebase myself):

- what I see is that you have a Resolver in `LoginConnectionService.java` which _doesn't_ use a custom DNS resolver for IPv6 preference
- inside `HttpConnectionPoolBuilder.java` - I've reverted the custom DNS resolver

So instead of causing there to be two different resolver implementations - they're actually now the same? I also don't see any reference to a resolver within the `http` directory.